src/Accounts/Accounts/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 
 ## Upcoming Release
+* Logging in by Web Account Manager (WAM) will be enabled by default when login interactively at 5/21.
```

Comment:
1. We could be more informative here. Suggestion:
Preannouncement: the default interactive login experience will change from browser-based to WAM-based (Web Account Manager) on supported platforms. This will take effect from the release of May 21st.

BTW, the date format "a/b" can sometimes be confusing so we should avoid using it.